### PR TITLE
Adds an Edit Profile link to the admin bar

### PR DESCRIPTION
**Pull request by JarrydLong**
- Navigate to your directory page
- Click on a user's profile
- You can now edit that specific user's profile

image

### CODE REVIEW

File: pmpro-member-directory.php [hunk 1]
```diff
@@ -158,3 +163,39 @@ function pmpromd_plugin_row_meta($links, $file) {
...
+			$pu = $current_user;
+		else
+			$pu = false;
+
+		if( $pu ){
+
+			$edit_link = get_edit_user_link( $pu->ID );
+		    $admin_bar->add_menu( array(
+		        'id'    => 'pmpromd-edit-profile',
+		        'title' => __( 'Edit Profile', 'pmpro-member-directory' ),
```
Comment (sc0ttkclark) [inline pmpro-member-directory.php:190]:
```suggestion
		        'title' => esc_html__( 'Edit Profile', 'pmpro-member-directory' ),
```

File: pmpro-member-directory.php [hunk 2]
```diff
@@ -158,3 +163,39 @@ function pmpromd_plugin_row_meta($links, $file) {
...
+		        'href'  => $edit_link,
+		        'meta'  => array(
+		            'title' => __( 'Edit Profile', 'pmpro-member-directory' ),
```
Comment (sc0ttkclark) [inline pmpro-member-directory.php:193]: Just a note here -- the meta attributes are escaped via esc_attr() ultimately.

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Minor escape tweak here

File: pmpro-member-directory.php [hunk 3]
```diff
@@ -160,6 +165,42 @@ function pmpromd_plugin_row_meta($links, $file) {
...
+			$pu = get_user_by( 'slug', $wp_query->get( 'pu' ) );
+		elseif( !empty( $current_user->ID ) )
+			$pu = $current_user;
+		else
+			$pu = false;
+
+		if( $pu ){
+
+			$edit_link = get_edit_user_link( $pu->ID );
+		    $admin_bar->add_menu( array(
```
Comment (ideadude) [inline pmpro-member-directory.php:188]:
```suggestion
		    // Note: The title is escaped here. The URL and meta attributes are escaped by the WP Admin Bar class.
		    $admin_bar->add_menu( array(
```

**Pull request merged**